core/src/main/scala/akka/kafka/javadsl/Committer.scala

```diff
@@ -21,10 +23,43 @@ object Committer {
   def flow[C <: Committable](settings: CommitterSettings): Flow[C, Done, NotUsed] =
     scaladsl.Committer.flow(settings).asJava
 
+  /**
+   * Batches offsets and commits them to Kafka, emits `CommittableOffsetBatch` for every committed batch.
+   */
+  def batchFlow[C <: Committable](settings: CommitterSettings): Flow[C, CommittableOffsetBatch, NotUsed] =
```

Comment:
This was lacking in the Java DSL, I'm tempted to require `CommittableOffset` here as well, but that would differ from the Scala DSL.

Reply:
Yes, I think it is good to keep them the same.